lib/order.rb

```diff
@@ -0,0 +1,45 @@
+require_relative 'menu'
+
+class Order
+
+  attr_reader :menu, :total_order, :bill
+
+  def initialize(menu = Menu.new)
+    @total_order = []
+    @menu = menu
+  end
+
+  def input
+    puts "Enter item number:"
```

Comment:
This works but is there a way to use an array so the user can input multiple items to their order at once?